diff --git a/external/faiss_sources/CMakeLists.txt b/external/faiss_sources/CMakeLists.txt
index dbc694b..b637b3e 100644
--- a/external/faiss_sources/CMakeLists.txt
+++ b/external/faiss_sources/CMakeLists.txt
@@ -7,7 +7,7 @@
 cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
 
 project(faiss
-  VERSION 1.7.2
+  VERSION 1.6.4
   DESCRIPTION "A library for efficient similarity search and clustering of dense vectors."
   HOMEPAGE_URL "https://github.com/facebookresearch/faiss"
   LANGUAGES CXX)
